fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -2386,6 +2387,14 @@ private void createOlapOrLakeTable(Database db, CreateTableStmt stmt) throws Ddl
                 } else {
                     throw new DdlException("Unsupported partition method: " + partitionInfo.getType().name());
                 }
+                // if binlog_enable is true when creating table,
+                // then set binlogAvailableVersion without statistics through reportHandler
+                if (olapTable.containsBinlogConfig() && olapTable.enableBinlog()) {
```

Comment:
just check if we have enabled binlog or not?

Reply:
done